config_vsc.py

```diff
@@ -330,22 +343,12 @@
                     'append': False,
                 },
             ],
-            'handlers_perflog': [
-                {
-                    'type': 'filelog',
-                    'prefix': '%(check_system)s/%(check_partition)s',
-                    'level': 'info',
-                    'format': '%(check_job_completion_time)s ' + perf_logging_format,
-                    'append': True,
-                },
-                {
-                    'type': 'syslog',
-                    'address': '/dev/log',
-                    'level': syslog_level,
-                    'format': perf_logging_format,
-                    'append': True,
-                },
-            ],
         }
     ],
-}
+    'modes': [
+        {
+            'name': 'standard',
```

Comment:
this mode is specific for antwerpen, so rename to e.g. `mode_antwerpen`?

Reply:
ok